data/hai/unfettered 2 war preparations.txt

```diff
@@ -93,13 +94,33 @@ mission "<uhai> 10: Belt Materials"
 		log `Gathered materials for the Unfettered shipyards. They're building sizable ships here.`
 		outfit "Lead" -50
 		outfit "Silver" -20
-		payment 1000000
+		payment 500000
 		conversation
 			`You bring your cargo to the local Unfettered shipyards, and see huge ships being built in the distance. You also notice that not all materials are brought to the shipyards, some are simply sent to more rural areas. You may want to check the shipyards, later on.`
 			`	One of the Unfettered team leaders comes to speak to you, "My greatest thanks for these materials, they will be crucial in repairing our houses, probably more so than for our ships. Please, if you have the time, try to bring us more materials, we post missions on the job board when we need them the most. In the meantime, take these <payment> as a token of my gratitude."`
 
 
 
+mission "<uhai> 10: Titanium"
+	name `Gather Titanium`
+	description `Mine 40 tons of Titanium from <waypoints> for <payment>.`
+	source "Firelode"
+	waypoint "Ik'kara'ka"
+	destination "Darkcloak"
+	to offer
+		has "salary: Unfettered"
+		"reputation: Wanderer" >= 0
+	on offer
+		dialog "Do you want to go get Titanium from <waypoints> as Mera suggested?"
```

Comment:
Why do you ask a question in a popup window, where you can't have choices?

Reply:
when a dialog is present in an on offer it acts as a decline/accept option